tests/workout.test.js

```diff
@@ -27,39 +34,124 @@ describe('Workout Model', () => {
     time_per_exercise: 60
   }
 
-  describe('Workout validations: ', () => {
+  describe('Model: ', () => {
+
+    describe('Workout validations: ', () => {
+
+      it('successfully creates a valid workout', () => {
+        return Workout.create(validWorkout)
+          .then(createdWorkout => {
+            expect(createdWorkout).to.contain(validWorkout)
+          })
+          .catch(err => console.error(err))
+      })
+
+      it('reports a validation error for invalid workout entries', () => {
+        return Workout.create(invalidWorkout)
+          .then(error => {
```

Comment:
Where did you define "invalidworkout" is it because its null that it defaults to invalid? (saw where you defined the valid one so not sure if there has to be an invalid one also)